src/applications/gi/components/heading/Breadcrumbs.jsx

```diff
@@ -38,26 +41,19 @@ class Breadcrumbs extends React.Component {
           </a>,
         );
       }
+      crumbs.push(
+        <Link to={`/profile/${facilityCode}`} key="result-detail">
```

Comment:
I don't think this is ever an active link, but it should be relative (i.e. not having the leading `/`)